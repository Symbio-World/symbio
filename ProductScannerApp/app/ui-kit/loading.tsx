import React from 'react'
import { View, ActivityIndicator } from 'react-native'
import { t } from 'react-native-tailwindcss'
import { colors } from './theme'

export const Loading = () => {
  return (
    <View style={[t.flex1, t.justifyCenter]}>
      <ActivityIndicator size="large" color={colors.primary} />
    </View>
  )
}
