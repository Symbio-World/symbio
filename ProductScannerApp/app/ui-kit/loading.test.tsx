import 'react-native'
import React from 'react'
import { create } from 'react-test-renderer'

import { Loading } from './loading'

describe('Loading', () => {
  it('renders correctly', () => {
    const tree = create(<Loading />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
