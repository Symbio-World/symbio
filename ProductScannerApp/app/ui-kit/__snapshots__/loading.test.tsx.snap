// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#4CAF50"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
